Payments/Integration/01_ManageConnection.md

```diff
@@ -0,0 +1,140 @@
+    The *Edit connection* view is displayed.
+
+    > [Info] Depending on the driver, the *Credentials* as well as the *Settings* tab are displayed in the *Edit connection* view.
+
+    ![Connection credentials](../../Assets/Screenshots/Payments/Settings/EditConnectionCredentials.png "[Connection credentials]")
+
+2. Edit the desired data of the connection in the corresponding fields in the *Credentials* tab.
+
+3. If necessary, click the *Settings* tab and edit the desired settings of the connection in the corresponding fields.
+
+4. Click the [SAVE] button.   
+    The *Checking credentials...* notice is displayed.
+    ![Checking credentials](../../Assets/Screenshots/Payments/Settings/CheckingCredentials.png "[Checking credentials]")
+    The *Edit connection* view is automatically closed when the changes have been saved. The *CONNECTIONS* view is displayed again.
+
+## Disable a PSP connection
+Once a connection has been established, you cannot delete it because of potential dependencies in the existing database. Nevertheless, you can disable the connection if you no longer use it. You can disable several active connections at a time.
+
+#### Prerequisites
+
+- At least one connection has been established, see [Create a connection](#create-a-connection).
+- The connection is active.
+
+
+#### Procedure
+
+*Payments > Settings > Tab CONNECTIONS*
+
+![Connections](../../Assets/Screenshots/Payments/Settings/ChangeConnections.png "[Connections]")
+
+1. Select the checkbox of the connection you want to disable in the list of connections.   
+    The editing toolbar is displayed above the list of connections.
+
+2. Click the [DISABLE] button in the editing toolbar.   
+    The connection has been deactivated.
+
+3. Click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button to update the list of connections.   
+    The status of the disabled connection in the *Status* column has switched to **Inactive**.
+
+## Enable a PSP connection
+Once a connection has been established, you can disable it if you no longer need it. If you have done this by mistake, you can enable the connection again. You can enable several active connections at once.
+
+#### Prerequisites
+
+- At least one connection has the status *Inactive*.
+
+
+#### Procedure
+
+*Payments > Settings > Tab CONNECTIONS*
+
+![Connections](../../Assets/Screenshots/Payments/Settings/ChangeConnections.png "[Connections]")
+<!----Stefan, kannst Du mir einen Screenshot machen?Eigentlich brauche ich nur einen grünen Enable button --->
+
+1. Select the checkbox of the connection you want to enable in the list of connections.   
+    The editing toolbar is displayed above the list of connections.
+
+2. Click the [ENABLE] button in the editing toolbar.   
+    The connection has been activated.
+
+3. Click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button to update the list of connections.   
+    The *Checking credentials...* notice is displayed. 
```

Comment:
Stimmt das so hier?

Reply:
Ja, auch in Sandbox kommt die Meldung. Aber nur für ein paar Millisekunden